projects/angular/src/emphasis/alert/STYLES.md

```diff
@@ -108,6 +108,12 @@
 | --clr-alert-sm-line-height                       | Alert small line height                                           |
 | --clr-alert-app-padding                          | App level alert padding                                           |
 | --clr-alert-items-gap                            | Gap between items in the alert                                    |
+| --clr-alert-sm-line-height                       | Small alert line height                                           |
+| --clr-alert-sm-font-size                         | Small alert font size                                             |
+| --clr-alert-sm-letter-spacing                    | Small alert letter spacing                                        |
+| --clr-alert-sm-font-weight                       | Small alert font weight                                           |
+| --clr-alert-sm-icon-size                         | Small alert icon size                                             |
+| --clr-alert-sm-item-text-align-items             | Small alert text align items                                      |
```

Comment:
We don't need to write down tokens which are temporary `ATM` and are about to be removed.

Reply:
`clr-alert-sm-item-text-align-items` is removed in commit 3b37d84